src/main/java/org/tugraz/sysds/runtime/util/LocalFileUtils.java

```diff
@@ -65,6 +65,7 @@
 	public static final String CATEGORY_RESULTMERGE  = "resultmerge";
 	public static final String CATEGORY_WORK         = "work";
 	public static final String CATEGORY_CODEGEN      = "codegen";
+	public static final String CATEGORY_LINEAGE		 = "lineage";
```

Comment:
The formatting is off - please use spaces inside code lines (but tabs at the beginning)